earthmover/yaml_parser.py

```diff
@@ -28,14 +28,7 @@ def construct_mapping(self, node, deep=False):
         :return:
         """
         mapping = super().construct_mapping(node, deep=deep)
-
-        #
-        for k, v in mapping.copy().items():
-            del mapping[k]
-            if isinstance(v, str):
-                mapping[os.path.expandvars(k)] = os.path.expandvars(v)
-            else: mapping[os.path.expandvars(k)] = v
-
+        
```

Comment:
I will want to chat with you to learn more about why this section is no longer applicable (mostly because I don't understand it myself).

Reply:
It's no longer needed because [this line](https://github.com/edanalytics/earthmover/blob/2d54886a39184760d9039c607693f60e79785c35/earthmover/earthmover.py#L153) expands vars in the entire template pre-YAML-parse, so no need to expand them in each key and (string) value post-Yaml-parse.

I still want to use the custom YAML parser class though, to get the associated line numbers.